cmd/lotus-storage-miner/sealing.go

```diff
@@ -253,3 +261,47 @@ var sealingSchedDiagCmd = &cli.Command{
 		return nil
 	},
 }
+
+var sealingAbortCmd = &cli.Command{
+	Name:      "abort",
+	Usage:     "Abort a running job",
+	ArgsUsage: "[call id]",
```

Comment:
Wanna call this `callid` instead, it kinda sounds like it takes 2 separate arguments `call` and `id` otherwise,